### PR TITLE
CLOUD-637 - cr-minimal introduce

### PR DESCRIPTION
**Pull request by spron-in**
[CLOUD-637](https://jira.percona.com/browse/CLOUD-637) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| init-deploy | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| limits | passed |
| scaling | passed |
| arbiter | passed |
| scheduled-backup | passed |
| security-context | passed |
| self-healing-chaos | passed |
| demand-backup-sharded | passed |
| service-per-pod | passed |
| rs-shard-migration | passed |
| liveness | passed |
| operator-self-healing | passed |
| upgrade | passed |
| smart-update | passed |
| upgrade-sharded | passed |
| version-service | passed |
| operator-self-healing-chaos | passed |
| pitr | passed |
| users | passed |
| data-sharded | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/736/commits/bf37df15aa4957b39935fab601d50a4b43cc90de
image: `perconalab/percona-server-mongodb-operator:PR-736-bf37df15`

File: deploy/cr-minimal.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  crVersion: 1.9.0
+  image: percona/percona-server-mongodb:4.4.6-8
+  allowUnsafeConfigurations: true
+  updateStrategy: SmartUpdate
+  upgradeOptions:
+    versionServiceEndpoint: https://check.percona.com
+    apply: 4.4-recommended
+    schedule: "0 2 * * *"
+  secrets:
+    users: minimal-cluster-secrets
```
Comment (delgod) [inline deploy/cr-minimal.yaml:17]: can we remove `image`, `updateStrategy`, `versionServiceEndpoint`, `secrets.users`?
```suggestion
  crVersion: 1.9.0
  allowUnsafeConfigurations: true
  upgradeOptions:
    apply: 4.4-recommended
    schedule: "0 2 * * *"
```

**Review by delgod — CHANGES_REQUESTED** [review]

File: deploy/cr-minimal.yaml (hunk 1 above)
Comment (spron-in) [reply deploy/cr-minimal.yaml]: Uploaded new version. It is as minimal as it gets. 
Removing secrets is possible, but then the weirdly named secret is created: percona-server-mongodb-users

lets keep it to be explicit and ease the docs.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| init-deploy | passed |
| storage | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| limits | passed |
| scheduled-backup | passed |
| arbiter | passed |
| scaling | passed |
| self-healing-chaos | passed |
| security-context | passed |
| demand-backup-sharded | passed |
| service-per-pod | passed |
| rs-shard-migration | passed |
| liveness | passed |
| operator-self-healing | passed |
| smart-update | passed |
| upgrade | passed |
| operator-self-healing-chaos | passed |
| upgrade-sharded | passed |
| version-service | passed |
| pitr | passed |
| users | passed |
| data-sharded | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/736/commits/935e4a750619f599799adb83e290b24c23f30630
image: `perconalab/percona-server-mongodb-operator:PR-736-935e4a75`

**Review by delgod — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| one-pod | passed |
| limits | passed |
| self-healing | passed |
| monitoring-2-0 | passed |
| scaling | passed |
| self-healing-chaos | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| service-per-pod | passed |
| rs-shard-migration | passed |
| operator-self-healing-chaos | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| data-sharded | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-sharded | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| pitr | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/736/commits/935e4a750619f599799adb83e290b24c23f30630
image: `perconalab/percona-server-mongodb-operator:PR-736-935e4a75`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| init-deploy | passed |
| version-service | passed |
| limits | passed |
| storage | passed |
| users | passed |
| scaling | passed |
| self-healing | passed |
| data-sharded | passed |
| security-context | passed |
| self-healing-chaos | passed |
| non-voting | passed |
| rs-shard-migration | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-sharded | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| pitr | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/736/commits/52f1931106e0d261a0e68595895d1d0cc3b9889e
image: `perconalab/percona-server-mongodb-operator:PR-736-52f19311`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| storage | passed |
| init-deploy | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| limits | passed |
| scaling | passed |
| scheduled-backup | passed |
| arbiter | passed |
| security-context | passed |
| self-healing-chaos | passed |
| service-per-pod | passed |
| rs-shard-migration | passed |
| demand-backup-sharded | passed |
| operator-self-healing | passed |
| upgrade | passed |
| liveness | passed |
| operator-self-healing-chaos | passed |
| upgrade-sharded | passed |
| smart-update | passed |
| pitr | passed |
| version-service | passed |
| users | passed |
| data-sharded | passed |
| pitr-sharded | passed |
| non-voting | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/736/commits/4d10ada937e01663673d766da7590df15dab9076
image: `perconalab/percona-server-mongodb-operator:PR-736-4d10ada9`

**Review by delgod — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**